openapi/sms/v1-review/common.yaml

```diff
@@ -0,0 +1,117 @@
+components:
+  schemas:
+    ReceivedAt:
+      type: object
+      properties:
+        receivedAt:
+          type: string
+          format: date-time
+          example: '2017-07-21T17:32:28Z'
+      required:
+        - receivedAt
+    EmarsysMessageId:
+      type: object
+      properties:
+        emarsysMessageId:
+          type: string
+          example: 'WI0ksjocu3b8W8kQU26M'
+          description: 'Unique identifier for an individual message send. This is used to map outbound message to the delivery report.'
+      required:
+        - emarsysMessageId
+    Error:
```

Comment:
The error does not conform to https://github.tools.sap/northstar/integration-kernel-services/blob/master/ap-iks-api-guidance/rest-api-harmonization/1.4.md#error-response-common-structure. 

Is it ok?

Reply:
good point, though we already have CustDev using this, we will make a ticket to try and conform in the future. nice find